domain/src/main/java/me/androidbox/domain/authentication/NetworkResponseState.kt

```diff
@@ -0,0 +1,9 @@
+package me.androidbox.domain.authentication
+
+sealed interface NetworkResponseState<out T> {
```

Comment:
I wouldn't name it NetworkResponseState, since this is in the domain, and you don't want the domain to know exactly from where the response is (it can be db also)

I'd name it to something like
```
Resource
```

Reply:
@MKiperszmid 
Renames to ResponseState